tests/e2e/system_test/test_cr_resources_validation.py

```diff
@@ -43,36 +48,49 @@ def finalizer():
                     run_cmd(f"rm {temp_file}")
 
             if self.object_name_to_delete != "":
-                res = run_oc_command(cmd=f"delete {self.object_name_to_delete}")
-                assert (
-                    ERRMSG not in res[0]
-                ), f"Failed to delete network fence resource with name: {self.object_name_to_delete}, got result: {res}"
+                res = run_oc_command(
+                    cmd=f"delete -n {self.namespace} {self.object_name_to_delete}"
+                )
+                assert ERRMSG not in res[0], (
+                    f"Failed to delete {self.object_kind_to_delete} resource with name: {self.object_name_to_delete}, "
+                    f"got result: {res}"
+                )
+
+            for pvc_obj in self.pvc_objs_to_delete:
+                pvc_obj.delete()
 
         request.addfinalizer(finalizer)
 
     def cr_resource_not_editable(
-        self, cr_object_kind, yaml_name, non_editable_patches, editable_patches
+        self,
+        cr_object_kind,
+        cr_resource_yaml,
+        non_editable_patches,
+        editable_patches,
+        namespace="openshift-storage",
     ):
         """
         Test that cr object is not editable once created
 
         Args:
             cr_object_kind (str): cr object kind
-            yaml_name (str): name of the yaml file from which the object is to be created
+            cr_resource_yaml (str): full path  of the yaml file from which the object is to be created
             non_editable_patches (dict, of str: str): patches to be applied by 'oc patch' command. These patches should
                     have no effect. If such a patch is sucessfully applied, the test should fail
             editable_patches (dict, of str: str): patches to be applied by 'oc patch' command. These patches should
                     have an effect. If such a patch is not sucessfully applied, the test should fail
+            namespace (str): namespace in which cr object should be created
 
         """
-        cr_resource_yaml = os.path.join(constants.TEMPLATE_CSI_ADDONS_DIR, yaml_name)
-        res = run_oc_command(cmd=f"create -f {cr_resource_yaml}")
+        res = run_oc_command(cmd=f"create -n {namespace} -f {cr_resource_yaml}")
```

Comment:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/helpers/performance_lib.py#L111

Why not using the namespace parameter from run_oc_command? The same in other function calls as well.

Reply:
You are right. Fixed it, thank you!